samples/ts/21.react-media-template/.gitignore

```diff
@@ -0,0 +1,28 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
You might also need to add a `.eslintignore` 😕 error might come up when running `doctor`